nfd-worker-daemonset.yaml.template

```diff
@@ -187,4 +195,25 @@ data:
     #            vendor: ["15b3"]
     #            device: ["1014", "1017"]
     #          loadedKMod : ["vendor_kmod1", "vendor_kmod2"]
+    #    - name: "feature.by.hostname"
+    #      value: customValue
+    #      matchOn:
+    #        - hostname: ["worker-0", "my-.*-node"]
 ### <NFD-WORKER-CONF-END-DO-NOT-REMOVE>
+---
+## Example for more custom configs in an additional configmap (3/3)
+#apiVersion: v1
+#kind: ConfigMap
+#metadata:
+#  name: more-custom-configs
+#  namespace: node-feature-discovery
+#data:
+## Filename doesn't matter, and there can be multiple. They just need to be unique across all additional ConfigMaps
+#  custom.conf: |
+#  - name: "more.kernel.features"
```

Comment:
just a yaml lint here, line 213 and below need to be moved to the right 2 spaces.
when moved to a single file, you get `error: error parsing custom-configs.yaml: error converting YAML to JSON: yaml: line 9: did not find expected key` meaning everything from line 9 is miss indented